azurelinuxagent/common/protocol/wire.py

```diff
@@ -970,8 +976,9 @@ def send_request_using_appropriate_channel(self, direct_func, host_func):
         ret = secondary_channel()
         if ret is not None:
             HostPluginProtocol.is_default_channel = not HostPluginProtocol.is_default_channel
-            logger.info("Default channel changed to {0}.", "host" if HostPluginProtocol.is_default_channel else "direct")
-
+            message = "Default channel changed to {0}.".format("host" if HostPluginProtocol.is_default_channel else "direct")
```

Comment:
Being very nitpicky here, but would it make more sense to say "HostGA" instead of "host"?
(either works for me!)

Reply:
Sure, HostGA is more specific.